### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by kaitcham**
For this project, I followed these [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md)

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 <% @users.each do |user| %>
     <%= render user %>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<button><%= link_to "Add Post", new_user_post_path(@current_user) %></button>
```
Comment (Rustamxon7) [inline app/views/users/index.html.erb:7]: Your app is not running properly:

Screen Shot 2022-07-16 at 5 30 44 AM

- [ ] Kindly make sure all the functionalities are completed as requirement and working properly as well.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
```
Comment (Rustamxon7) [inline app/controllers/application_controller.rb:2]: - [ ] [OPTIONAL] Kindly add the MIT license to your project.

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @kaitcham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (kaitcham) [reply app/views/users/index.html.erb:7]: Hi @Rustamxon7  Thanks for review my project but can you please clarify the changes you are requesting from me, I don't think there are in the requirements

**Review by EricMbouwe — APPROVED** [review]
Hi @kaitcham, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- ⚡ Make requested changes 
- ⚡Forms 
## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kaitcham** [comment]
@EricMbouwe Thanks for your amazing professional reviews

**Pull request merged**